### ISSUE TITLE
Is this code supported for Raspberry pi?

### ISSUE THREAD

**Issue by haykp** [Billwilliams1952/PiCameraApp#11]
I have not tested, but will this code work for Raspberry pi?
If not,  is is possible to add support?

**Comment by Billwilliams1952** [comment]
It’s written specifically for the RPI.

**Comment by bogaskilop** [comment]
can you  write  any information what should be also installed in raspbian  so it will run as on fresh  installation it just not work :(

**Comment by robin-weaver** [comment]
> can you write any information what should be also installed in raspbian so it will run as on fresh installation it just not work :(

I agree, a dependencies section of the readme with the necessary commands would be great. Fantastic work on this btw!
